res_company_bank/models/res_company.py

```diff
@@ -0,0 +1,12 @@
+from odoo import fields, models
+
+
+class Company(models.Model):
+    _inherit = "res.company"
+
+    bank_info = fields.Char(
+        name="Bank",
+    )
+    bank_description = fields.Char(
```

Comment:
```suggestion
    description = fields.Char(
```
Sorry, discription is not for banks, it is for report.